README.md

```diff
@@ -1 +1,94 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top">SET UP LINTERS AND CREATE PULL REQUEST</a>
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="https://zbahati.github.io/Hello-Microverse/">visit website</a>
+
+**[Hello Microverse]**
+is a web application that help us to set up the linters for the project that we will use to test the project.
+
+#### this project have  one section that  is h1 with the message Hello Microverse.
+
+<!-- Features -->
+
+### Key Features <a name="https://zbahati.github.io/Hello-Microverse/">header-section features</a>
+
+
+- **[Test Linters of HTML and CSS ]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://zbahati.github.io/Hello-Microverse/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+> Describe how a new developer could make use of your project.
+to get start with this testing for new developer we could first start with installing the following things linters,lighthouse, .hintrc,stylelintrc.json,
+
+### Prerequisites
+you need to have editor for writing  the codes and
+you need to have the gitHub, and Git to test how is the project
+
+### install the neccessary command to run the project
+install webhint using npm init -y, then install webhint by npm install --save-dev hint@7.x
+install lighthouse with npm install -g lighthouse
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Use the command for clone the report
+https://github.com/zbahati/Hello-Microverse.git
+
+### Deployment
+
+You can deploy this project using GitHub by going on
+1.Setting
+2.pages
+3. publish the page
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Bahati Zirimwabagabo <a name="https://github.com/zbahati">Zbahati</a>
+
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/zbahati)
+- Twitter: [@twitterhandle](https://twitter.com/@b2_bahati)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/zirimwabagabo-bahati)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+ [ ] **[html-makeup-feature]**
+- [ ] **[style-feature]**
+- [ ] **[Fixing-bugs-Feature]**
+
+## 🤝 Contributing <a name="contributing"></a>
+Microverse Reviewers help me to Add new Features,new  neccessary files and change or delete unnecessary files
+Microverse students help me to setup the project and fixing some issues regarding the setup
+
+#### ⭐️ Show your support <a name="support"></a>
+Thank you for using our linter setup project!
+Your support is greatly appreciated and helps us continue to improve this project.
+Here are three ways you can support us:
+
+Star the project on GitHub to show your appreciation
+Contribute to the project by submitting bug reports or feature requests
+Spread the word by telling your friends and colleagues about our project
+Thank you for your support!
+
+
```

Comment:
- [ ] Please include `Acknowledment section ` in your README to give credit where credit is due.